Formula/s/steamguard-cli.rb

```diff
@@ -0,0 +1,24 @@
+class SteamguardCli < Formula
+  desc "CLI for steamguard"
+  homepage "https://github.com/dyc3/steamguard-cli"
+  url "https://github.com/dyc3/steamguard-cli/archive/refs/tags/v0.12.5.tar.gz"
+  sha256 "fce353371010cbc298e6dc2d7063742178639eb90127b752d85c3cf92a46661d"
+  license "GPL-3.0-only"
```

Comment:
https://github.com/dyc3/steamguard-cli/blob/master/Cargo.toml#L14

```suggestion
  license "GPL-3.0-or-later"
```